src/cls/rgw/cls_rgw.cc

```diff
@@ -1780,7 +1785,8 @@ static int rgw_bucket_link_olh(cls_method_context_t hctx, bufferlist *in, buffer
       unmod.tv_nsec = 0;
     }
     if (mtime >= unmod) {
-      return 0; /* no need tof set error, we just return 0 and avoid
+      return 0;
+     /* no need tof set error, we just return 0 and avoid
```

Comment:
I realize you didn't create this misspelling, but since you're editing here anyway maybe change "tof" to "to".